snap/hooks/configure

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
+#!/bin/sh
+'''exec' "$(dirname -- "$(realpath -- "$0")")"/../../bin/'python3' "$0" "$@"
```

Comment:
Will other snaps need to do this for their hooks once we have a uv plugin?

Reply:
Yes, though I'm not entirely sure why. But `/usr/bin/env` was finding `/usr/bin/python3` rather than `/snap/charmcraft/current/bin/python3`. (Or it was to do with something else I did and it would work now. Will check.)

Reply:
UPDATE: It was something else I did. I'll revert these changes.